healpix_alchemy/tests/test_regions.py

```diff
@@ -0,0 +1,103 @@
+import re
+
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+from astropy import units as u
+from astropy.utils.data import get_readable_fileobj
+from mocpy import MOC
+import numpy as np
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+
+from .. import Region, Tile
+
+
+Base = declarative_base()
+
+
+class Field(Region, Base):
+    __tablename__ = 'fields'
+
+    field_id = Column(
+        Integer,
+        primary_key=True)
+
+    tile_class = lambda: FieldTile  # pylint: disable=E731 # noqa
```

Comment:
Why both the pylint and noqa directives?

Reply:
unnecessary! removed.